Analysis/interface/TauTuple.h

```diff
@@ -162,6 +164,8 @@
     TAU_VAR(Int_t, index) /* index of the tau */ \
     TAU_VAR4(Float_t, pt, eta, phi, mass) /* 4-momentum of the tau */ \
     TAU_VAR(Int_t, charge) /* tau charge */ \
+    /* Tau selection type of TagAndProbe selector */ \
+    TAU_VAR (Int_t, selectionType) /* 1 Muon tag, 2 Ele tag, 3 High MET, 4 Pt ordered, 5 gen based */ \
```

Comment:
it is not `TAU_VAR`, but rather `VAR` as it was before. It could be that HPS tau is not reconstructed.